crates/oxc_parser/src/ts/statement.rs

```diff
@@ -356,13 +356,15 @@ impl<'a> Parser<'a> {
         false
     }
 
-    pub fn parse_ts_declare_statement(&mut self) -> Result<Statement<'a>> {
+    pub fn parse_ts_declaration_statement(&mut self) -> Result<Statement<'a>> {
         let declaration = self.parse_declaration_clause()?;
         Ok(Statement::Declaration(declaration))
     }
 
     pub fn parse_declaration_clause(&mut self) -> Result<Declaration<'a>> {
         let has_ambient = self.ctx.has_ambient();
+
+        // todo(yangchen): Handle all modifiers together.
         let declare = self.eat(Kind::Declare);
```

Comment:
In the future we can eat modifiers here and maybe extend the context to store them.

Reply:
Can you remove the todo comment and create an issue instead?

Reply:
Done